pydundas/tests/rest/test_constant.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+from pydundas import Api
+from pydundas.rest.constant import ConstantIdError, ConstantNameError
+
+
+class TestConstant(unittest.TestCase):
```

Comment:
I realize that it's hard to test the actual values of the constants because they may be changed with a modification of the api on Dundas side, but I'd argue that this test is meaningless as it is now, because it does not actually test the class under test.
I'd much rather see this code test the _actual_ values of the api. Let it break when the constants are changed because of a modification to the code base because of a change in Dundas. That would mean someone forgot to check (and fix) the tests after modifying the constants.

If you're only interested in testing the mechanism of fetching a constant, some kind of mocking would be more appropriate. That would actually separate the testing of the mechanism from the data, which is always a good principle.

Reply:
The sad truth is that there is no way to get the constants via the rest api, so no test possible for changes.